### ISSUE TITLE
Basyx Updater - One datasource to several AAS properties

### ISSUE THREAD

**Issue by dgarcoe** [eclipse-basyx/basyx-databridge#1]
Hi,

We are having some trouble mapping a JSON with several values coming from only one datasource to several datasinks. I'll explain it better through a simple use case:

We have some JSON like the next one: `{"A":1, "B":2, "C":10}` being published in a topic `/data`

We would like to match each variable value with a property in an AAS submodel. 

We configured only one datasource and a routes file as the following one but it doesn't seem to be viable:

```
[
	{
		"datasource": "mqtt",
		"transformers": ["jsonataA", "jsonataB", "jsonataC"],
		"datasinks": ["ConnectedSubmodel/ConnectedPropertyA", "ConnectedSubmodel/ConnectedPropertyB", "ConnectedSubmodel/ConnectedPropertyC"]
	}
]
```

Is this possible?

**Comment by aaronzi** [comment]
This is now implemented with #308

**Comment by michelde** [comment]
@aaronzi: Would be nice to see this feature in the wiki documented. Can't find it?!
